tasks/kmt.py

```diff
@@ -58,6 +60,59 @@ def gen_config(
     vmconfig.gen_config(ctx, stack, vms, sets, init_stack, vcpu, memory, new, ci, arch, output_file)
 
 
+@task(
+    help={
+        "pipeline": "Pipeline ID to use",
+        "stack": "Name of the stack within which to generate the configuration file",
+        "vcpu": "Comma separated list of CPUs, to launch each VM with",
+        "memory": "Comma separated list of memory to launch each VM with. Automatically rounded up to power of 2",
+        "new": "Generate new configuration file instead of appending to existing one within the provided stack",
+        "init-stack": "Automatically initialize stack if not present. Equivalent to calling 'inv -e kmt.create-stack [--stack=<stack>]'",
+    }
+)
+def gen_config_from_ci_pipeline(
```

Comment:
What do you think about overloading `gen-config` such that when the user provides the `pipeline` argument, this function runs? That way we can continue having a single task for generating the configurations.